README.md

```diff
@@ -70,6 +70,13 @@ cd drupal-admin-ui
 yarn start
 ```
 
+### React AXE
+
+Due to outstanding performance issues, `react-axe` is behind a flag. To enable the assessment provided by `react-axe` pass an environment variable when starting the application.
+
+```
+REACT_APP_AXE=true yarn start
```

Comment:
Interesting!

Should we maybe set this environment variable as part of nightwatch? I'm wondering whether we want to log the errors as part of testing or not.

Reply:
Yes. We could probably enable this on the CI environment. However, `react-axe` appears to add a pretty significant performance penalty which might have been some of the timeout nightwatch issues.